templates/Segment_Image_Data_GoogleDriveExample.ipynb

```diff
@@ -0,0 +1,450 @@
+   "metadata": {
+    "tags": [
+     "load_fovs"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# either get all fovs in the folder...\n",
+    "if MIBItiff:\n",
+    "    fovs = io_utils.list_files(tiff_dir, substrs=MIBItiff_suffix)\n",
+    "else:\n",
+    "    fovs = io_utils.list_folders(tiff_dir)\n",
+    "\n",
+    "# ... or optionally, select a specific set of fovs manually\n",
+    "# fovs = [\"fov1\", \"fov2\"]\n",
+    "\n",
+    "print(fovs)\n",
+    "\n",
+    "# TODO: MIBItiff manual selection"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "nuc_mem_set"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# NOTE: at least one of nucs and mems must not be None\n",
+    "# nuclear channel name(s) (or nucs = None)\n",
+    "nucs = ['HH3']\n",
+    "\n",
+    "# membrane channel name(s) (or mems = None)\n",
+    "mems = None"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true,
+    "tags": [
+     "set_channels"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# load channels to be included in Mesmer data\n",
+    "channels = (nucs if nucs else []) + (mems if mems else [])\n",
+    "\n",
+    "# filter channels for None (just in case)\n",
+    "channels = [channel for channel in channels if channel is not None]"
+   ]
+  },
+  {
+   "source": [
+    "### Here we download the required image data directly from Drive.  No image data is saved locally!"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "load_data_xr"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "if MIBItiff:\n",
+    "    data_xr = load_utils.load_imgs_from_mibitiff(tiff_dir, mibitiff_files=fovs, channels=channels)\n",
+    "else:\n",
+    "    data_xr = load_utils.load_imgs_from_tree(tiff_dir, img_sub_folder=\"TIFs\", fovs=fovs, channels=channels)"
+   ]
+  },
+  {
+   "source": [
+    "### Note: If `deepcell_input_dir` is a local drive, the inputs for deepcell will be saved locally.  However, if `deepcell_input_dir` is a `GoogleDrivePath`, then the inputs will be automatically saved to the specified Drive folder.\n",
+    "\n",
+    "### In general, it's best to save 'mid-processed' data locally to avoid upload/download slow downs, and only upload data directly if it's 'final'.  That advice is ignored here for demonstration purposes"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "gen_input"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# generate and save deepcell input tifs\n",
+    "data_utils.generate_deepcell_input(data_xr, deepcell_input_dir, nucs, mems)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Upload files to Deepcell and download results\n",
```

Comment:
I think that's a vestigial H2 from an older version of the notebooks.  I'll H3 that and change Deepcell to Mesmer since I think that's the official ✨ _branding_ ✨  these days